setup.py

```diff
@@ -9,6 +9,7 @@
     exec(code, release_info)
 
 setup(
+    python_requires=">=3.9",
```

Comment:
duplicate arg `python_requires`, it is given later as: `python_requires=release_info["PYTHON_REQUIRES"]`
(it is read from info.py)

Reply:
Then we need the same in [populse_db](https://github.com/populse/populse_db/blob/39fb6c6f21e19ebc70746c50286e1dcd6f9b6da8/setup.py#L26-L44) and [capsul](https://github.com/populse/capsul/blob/e64de08460fe346581460eeafb0426be163205a8/setup.py#L24-L41), which lack this mechanism.

Unfortunately, 3rd party tools are unable to understand such config files. We'll have to invert the logic if we switch to `pyproject.toml`:
- the version and other metadata are defined in the config file, and the Python code has to retrieve metadata using dedicated modules such as [`importlib.metadata`](https://docs.python.org/3/library/importlib.metadata.html),
- instead of the version and other metadata are defined in the source code, so we're stuck with an executable `setup.py` instead of moving to a real config file.

Reply:
For now yes, we're a bit stuck, but we could move to `pyproject.toml` once our cmake-based tools can parse the `pyproject.toml` file. The main problem for me is that I don't know much about the `pyproject.toml` structure and contents, so I don't feel to do it myself.

Reply:
I can do that, but will need help with the cmake tools :-)

Reply:
We need to read version in both CMake and Python so we will have to make a `pyproject.toml` parser for this great CMake language. I can help but prepare to face code that will hurt your love love of good programming (all this is in brainvisa-cmake program).